gateway/1.4/modules/ROOT/pages/flex-gateway-k8-ingress-class.adoc

```diff
@@ -0,0 +1,61 @@
+= Configure Flex Gateway as an Ingress Controller in Local Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+For Kubernetes deployments in Local Mode, Flex Gateway typically acts as an ingress controller that manages external access to your Kubernetes cluster.
+
+To configure Flex Gateway as an ingress controller, you must configure an `IngressClass` resource and an `Ingress` resource. The `IngressClass` resource configures Flex Gateway as an ingress controller, and the `Ingress` resources configures the services Flex Gateway manages as an ingress controller.
+
+By default, Flex Gateways deployed on Kubernetes in Local Mode are defined as `IngressClass` resources.
+
+For a detailed explanation of Kubernetes `IngressClass` resources and `Ingress` resources, see https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes Ingress^].
+
+== Before You Begin
+
+Register a Flex Gateway instance in Local Mode and deploy this instance to a Kubernetes cluster. For guidance, see xref:flex-gateway-set-up.adoc[] or xref:flex-gateway-k8-getting-started.adoc[].
+
+== Ingress Class Parameters
+The Helm chart includes the following `IngressClass` parameters configured with the following values by default:
+
+[source,helm]
+----
+ingressClass.enabled: true
+ingressClass.name: "{releaseName}-{namespace}"
+ingressClass.setAsDefault: false
+----
+
+[cols="1,2,2"]
+|===
+| Parameter | Description | Possible values
+| `ingressClass.enabled` | Enables Flex Gateway as an `IngressClass` resource | `true` or `false`
+| `ingressClass.name` | The name of the `IngressClass` | Any string
+| `ingressClass.setAsDefault` | Sets Flex Gateway as the default `IngressClass` | `true` or `false`
+|===
+
+Setting `ingressClass.setAsDefault: true` defines the Flex Gateway instance as the default `IngressClass` resource for the cluster. The default `IngressClass` is the default ingress controller for all `Ingress` resources you create in the cluster. 
+
+If there is no default `IngressClass` or you do not want the default `IngressClass` resource as the controller for a specific resource, you must manually define the `IngressClassName` parameter in the `Ingress` resource.
+
+You can modify the default `IngressClass` parameters during or after the initial installation of the Helm chart. For more information about how to modify a Helm chart, see xref:flex-gateway-k8-change-helm-settings.adoc[]
+
+== Creating Ingress Resources
+
+After you xref:flex-local-reg-run-up#install-helm-chart-into-the-namespace[install a Helm chart into a namespace], you can confirm your new `IngressClass` resource is present by running the following command:
+
+[source,helm]
+----
+kubectl get ingressclasses
+----
+
+If your `IngressClass` resource is present, to create an ingress routing:
```

Comment:
```suggestion
Create an ingress routing:
```
I'm thinking it's implied that they should do this after verifying the resource is present?